core/server/api/canary/utils/serializers/output/member-signins.js

```diff
@@ -0,0 +1,7 @@
+module.exports = {
+    read(data, apiConfig, frame) {
+        frame.response = {
+            member_signins: [data]
```

Comment:
I'm unsure about best resource naming here. The `member_signin` made most sense to me at the time of writing (instead of `member_signin_url`) for couple reasons: 
- it's a resource that contains the `url` property and might get some more properties in the future (`expiration` date, just to name a possible one) 
- this name is more concise

The downside I see is that `signin` can be perceived as an action instead of a resource :thinking:

Reply:
This naming is also not corresponding with the endpoint name: `/members/:id/signin_url`( https://github.com/TryGhost/Ghost/pull/11607/files#diff-ac177f48adeea6d6a62218c79891c8faR103)

The endpoint can be renamed to `/members/:id/signin`, but this would lead to even more confusion as if it's an "action" or something.

Reply:
🤔 I am inclined towards going with `member_signin_url` compared to `member_signin` as former is definitely representing better what the resource and response is and feels like there are more downsides for going with more generic `member_signin` in long term.

Also imo even if the resource does get some extra properties in future like `expiration`, it still is denoting a property for the purpose of member's signin URL at high level and so `members_signin_url` should be fine for it, with a `url` and `expiration` property as part of it.

Reply:
I agree with @rishabhgrg here, let's stick to our naming convention and go with `GET /members/:id/signin_url/` with a response of `{member_signin_urls: [{url: '...', expires_at: '...', ...}]}`

Reply:
@kevinansfield for now the resource will return only the `url` and `member_id` properties, there is no clear usecase for `expires_at` at the moment, so will keep it to KISS rule. Totally agree with the rest, thanks for feedback!